src/core/storage.ts

```diff
@@ -345,4 +356,28 @@ export class Storage {
       return false
     }
   }
+
+  isCookiesEnabled(): boolean {
+    // Disabled by configuration
+    if (!this.options.cookie) {
+      return false
+    }
+
+    // Server can only assume cookies are enabled, it's up to the client browser
+    // to create them or not
+    if (process.server) {
+      return true
+    }
+
+    if (window.navigator.cookieEnabled) {
+      return true
+    } else {
+      // eslint-disable-next-line no-console
+      console.warn(
+        "[AUTH] Cookies is enabled in config, but browser doesn't" +
+          ' support it'
+      )
+    }
+    return false
```

Comment:
When the console warn is in the else, and the return out of it, it seems like the two aren't connected. It reads like a bug or a mistake.

Please either put the return in the else, or remove the else and just put the warn and return outside of the if/else structure. Both are functionally equivalent, and both group the two together so it's clear what's going on and why.